### PR TITLE
tproxyUpstreamConnMark

### PR DESCRIPTION
**Pull request by tomastigera**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: pkg/apis/projectcalico/v3/felixconfig.go [hunk 1]
```diff
@@ -698,6 +698,9 @@ type FelixConfigurationSpec struct {
 	// TPROXYPort sets to which port proxied traffic should be redirected.
 	// [Default: 16001]
 	TPROXYPort *int `json:"tproxyPort,omitempty" validate:"omitempty,gt=0,lte=65535"`
+	// TPROXYUpstreamConnMark tell Felix which mark is used by the proxy for its upstream
```
Comment (fasaxc) [inline pkg/apis/projectcalico/v3/felixconfig.go:701]: Typo: `s/tell/tells/`

**Review by fasaxc — APPROVED** [review]
One typo but LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**